### ISSUE TITLE
Predicting on overlapping clips

### ISSUE THREAD

**Issue by syunkova** [kitzeslab/opensoundscape#876]
Not really an issue - but should we add an overlap_time parameter (in addition to existing overlap_fraction) to predict() (and therefore AudioSplittingDataset, etc) so that the amount of overlap between clips could be indicated with time in seconds and not just a fraction?

**Comment by sammlapp** [comment]
Yeah we can add this. We could alternatively give "step_size" argument, ie 3s clip w/1s overlap has step size of 2. I think that's the parameter often used in other projects/apis.
